features/facebookImport/src/model/analyses/report/inactive-cards-summary.js

```diff
@@ -17,7 +17,7 @@ export default class InactiveCardsSummary {
         this._inactiveAnalysesData = inactiveAnalysesResults.map(
             ({ analysis, status }) => {
                 return {
-                    analysis: status.analysisClass.name,
+                    analysis: analysis.id,
```

Comment:
Can you expand a bit what you've done here? You're essentially switched some piece of information from one argument to the other? Is it because analysisClass has disappeared?

Reply:
Yes. In a result object we had both `analysis` and `status. analysisClass`. But we do not need to to store `analysisClass` in status, as it is redundant. We can take it from the analysis. All analyses have an id, which we can use. In all current analyses that is the name of the class.